src/components/Lend/LoanSearch/LoanSearchFilter.vue

```diff
@@ -85,24 +85,24 @@
 </template>
 
 <script>
-
 import {
-	fetchData,
-	filterGender,
-	filterSector,
+	// fetchData,
+	// filterGender,
+	// filterSector,
 	fetchSectors,
 	fetchCountryFacets,
-	filterCountry,
-	filterLoanTerm,
+	// filterCountry,
+	// filterLoanTerm,
 } from '@/util/flssUtils';
 import KvAccordionItem from '@/components/Kv/KvAccordionItem';
 import { mdiClose, mdiArrowRight } from '@mdi/js';
+import updateLoanSearchMutation from '@/graphql/mutation/updateLoanSearchState.graphql';
 import KvRadio from '~/@kiva/kv-components/vue/KvRadio';
 import KvCheckbox from '~/@kiva/kv-components/vue/KvCheckbox';
 import KvMaterialIcon from '~/@kiva/kv-components/vue/KvMaterialIcon';
 
 export default {
-	inject: ['apollo'],
+	inject: ['apollo', 'cookieStore'],
```

Comment:
Where is `cookieStore` used?

Reply:
Checkout https://github.com/kiva/ui/blob/main/src/api/apollo.js to see how cookieStore is a dependency of some of our Apollo links.